src/main/java/org/testobject/rest/api/RestClient.java

```diff
@@ -19,8 +19,7 @@
 
 public class RestClient implements Closeable {
 
-	public static final String REST_APPIUM_PATH = "/rest/v2/appium/";
-	public static final String REST_DEVICES_PATH = "/rest/devices/v1/";
+	public static final String REST = "/rest/";
```

Comment:
Is there a reason why you added this? It's unused

Reply:
yeah, the second one is not used. The first one should be replaced with "/rest/" after I decided to put the full path in the requests (e.g. https://github.com/saucelabs/testobject-java-api/pull/47/files#diff-aa94a3799c000888a25bd3996f3546e8R38) instead of doing some magic